cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -609,3 +613,45 @@ def test_guest_os_shutdown(appliance, provider, testing_vm_tools, ensure_vm_runn
         new_last_boot_time = view.entities.summary("Power Management").get_text_of("Last Boot Time")
         soft_assert(new_last_boot_time == last_boot_time,
                     "ui: {} should ==  orig: {}".format(new_last_boot_time, last_boot_time))
+
+
+@pytest.fixture(scope="function")
+def new_user(request, appliance):
+    user, user_data = _users(request, appliance, group="EvmGroup-vm_user")
+    new_user = appliance.collections.users.instantiate(
+        name=user[0].name,
+        credential=Credential(principal=user_data[0]["userid"], secret=user_data[0]["password"]),
+    )
+    yield new_user
+    if user[0].exists:
```

Comment:
We do not need this. `users` uses `create_skeleton` function, which will do the cleanup on its own. See this: https://github.com/ManageIQ/integration_tests/blob/master/cfme/rest/gen_data.py#L487

Reply:
@valaparthvi
 - I went through this fixture(https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/intelligence/reports/test_menus.py#L133). Can you plese confirm why are we [using this](https://github.com/ManageIQ/integration_tests/pull/9113#discussion-diff-311385009R626) in that fixture?

Reply:
Ah, that was a mistake on my part. I recently realized that `create_skeleton` deletes the entities it creates, I am working on removing that and other places where I might've used it.

Reply:
Conclusion: Since we cannot delete a user if they're tied to a request on 5.11, we decided to explicitly delete the user here.